crates/bevy_internal/src/default_plugins.rs

```diff
@@ -71,7 +71,57 @@ plugin_group! {
     ///
     /// [`DefaultPlugins`] contains all the plugins typically required to build
     /// a *Bevy* application which includes a *window* and presentation components.
-    /// For *headless* cases – without a *window* or presentation, see [`MinimalPlugins`].
+    /// For *headless* cases – without a *window* or presentation, see [`HeadlessPlugins`].
+    /// For the absolute minimum number of plugins needed to run a Bevy application, see [`MinimalPlugins`].
+}
+
+plugin_group! {
+    /// This plugin group will add all the default plugins for a headless (no *window* or rendering) *Bevy* application:
+    pub struct HeadlessPlugins {
+        bevy_app:::PanicHandlerPlugin,
+        bevy_log:::LogPlugin,
+        bevy_core:::TaskPoolPlugin,
+        bevy_core:::TypeRegistrationPlugin,
+        bevy_core:::FrameCountPlugin,
+        bevy_time:::TimePlugin,
+        bevy_transform:::TransformPlugin,
+        bevy_hierarchy:::HierarchyPlugin,
+        bevy_diagnostic:::DiagnosticsPlugin,
+        bevy_a11y:::AccessibilityPlugin,
```

Comment:
Accessibility might be less common than the rest, I wonder if it's worth including by default

Reply:
IMO this should also be excluded: it's very much tied to input handling and UI.